test/general.jl

```diff
@@ -9,32 +9,61 @@
 # Backend inference
 # ==============================================================================
 
+# minimal other table interface
+struct TestVec{T} <: AbstractArray{T,1}
+    data::Array{T,1}
+end
+Base.IndexStyle(::Type{A}) where {A<:TestVec} = Base.IndexCartesian()
+Base.size(A::TestVec) = size(getfield(A, :data))
+Base.getindex(A::TestVec, index::Int) = getindex(getfield(A, :data), index)
+Base.collect(::Type{T}, itr::TestVec) where {T} = TestVec(collect(T, getfield(itr, :data)))
+
+struct MinimalTable
+    data::Matrix
+    colnames::TestVec
+end
+
+Tables.istable(x::MinimalTable) = true
+Tables.columnaccess(::MinimalTable) = true
+Tables.columnnames(x::MinimalTable) = getfield(x, :colnames)
+Tables.columns(x::MinimalTable) = x
+Base.getindex(x::MinimalTable, i1, i2) = getindex(getfield(x, :data), i1, i2)
+Base.getproperty(x::MinimalTable, s::Symbol) = getindex(x, :, findfirst(==(s), Tables.columnnames(x)))
+
+table = MinimalTable([1 2; 3 4], TestVec([:a, :b]))
```

Comment:
Why this is required here?

Reply:
It's not necessary. I forgot to delete along with the empty lines at the end of the file while writing out some scratch code.